templates/.github/workflows/release-check.yml

```
@@ -1,7 +1,7 @@
 name: Release Checker
 on:
-  pull_request:
-    paths: [ 'version.json' ]
+  pull_request_target:
+    paths-ignore: [ '!version.json' ]
```

Can't we just use `paths`? I'll check before merging.
```suggestion
    paths: [ 'version.json' ]
```

---

Yes, we can